src/core/templatetags/uuid.py

```diff
@@ -0,0 +1,8 @@
+from django import template
+from uuid import uuid4
+
+register = template.Library()
+
+@register.simple_tag
+def short_uuid4():
```

Comment:
When truncating a UUID there is no way to guarantee it will be unique although a collision is super unlikely. Is there any need to shorten the UUID realistically?

Reply:
It was for readability of the rendered code, keeping in mind the low stakes of a clash for a convenience button, but I can expand it, so that it is safe to use in even high-stakes interfaces.